api/include/opentelemetry/context/runtime_context.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include "opentelemetry/context/context.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace context
+{
+// Provides a wrapper for propagating the context object globally. In order
+// to use either the threadlocal_context.h file must be included or another
+// implementation which must be derived from the RuntimeContext can be
+// provided.
+class RuntimeContext
+{
+public:
+  class Token
+  {
+  public:
+    bool operator==(const Context &other) { return context_ == other; }
```

Comment:
This makes it possible to compare a `Token` to a `Context`. I wonder if the argument should be of type `Token`?

Reply:
Yes, that was on purpose. There has not been a use case for token to token comparison.